docs/features/kubernetes/configuration.md

```diff
@@ -57,6 +57,10 @@ This is an array used to determine where to retrieve cluster configuration from.
 
 Valid cluster locator methods are:
 
+- [`config`](#config)
+- [`gke`](#gke)
+- [custom `KubernetesClustersSupplier`](#custom-kubernetesclusterssupplier)
```

Comment:
I added this list to prevent a bit of confusion I had while reading the docs. If it's out of scope for this MR I can remove this list again.